docs/kr.tree

```diff
@@ -249,10 +249,13 @@
                 </toc-element>
             </toc-element>
             <toc-element id="native-debugging.md"/>
-            <toc-element id="native-target-support.md" toc-title="Target support"/>
             <toc-element id="native-ios-symbolication.md"/>
-            <toc-element id="native-improving-compilation-time.md" toc-title="Improving compilation times"/>
-            <toc-element id="native-faq.md"/>
+            <toc-element toc-title="Resources and tips">
```

Comment:
The "Resources" term is too broad for me... I think we should find a better name for it.

Reply:
I'd try a general "reference" for now, maybe we'll find something better later